1.20/Dockerfile.fedora

```diff
@@ -59,19 +59,17 @@ COPY ./root/ /
 # writable as OpenShift default security model is to run the container under
 # random UID.
 RUN sed -i -f ${NGINX_APP_ROOT}/nginxconf-fed.sed ${NGINX_CONF_PATH} && \
-    chmod a+rwx ${NGINX_CONF_PATH} && \
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.default.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/src/nginx-start/ && \
     mkdir -p ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
     mkdir -p ${NGINX_LOG_PATH} && \
     mkdir -p ${NGINX_PERL_MODULE_PATH} && \
+    chmod a+rwx ${NGINX_CONF_PATH} && \
     chmod -R a+rwx ${NGINX_APP_ROOT}/etc && \
-    chmod -R a+rwx /var /run && \
+    chmod -R a+rwx ${NGINX_APP_ROOT}/src/nginx-start/  && \
     chmod -R a+rwx ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
-    chown -R 1001:0 ${NGINX_APP_ROOT} && \
-    chown -R 1001:0 /var && \
-    chown -R 1001:0 ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
```

Comment:
Are you sure about removing this line? 
This line is executed by this code https://github.com/sclorg/nginx-container/blob/master/1.20/s2i/bin/run#L9, which means, that `USER 1001` runs it.

Reply:
This is the same as issue you mentioned in the comment above.